data/migrations/20201109234734_seller_profie_and_item.js

```diff
@@ -9,7 +9,7 @@ exports.up = function (knex) {
       tb.text('description');
     })
     .createTable('buyer_profile', (tb) => {
-      tb.increments();
+      tb.string('id', 255).unique().notNullable().primary();
```

Comment:
For this line of code here. Doesn't table .increments() already take care of this?

Reply:
Because of the way Okta works they needed that specific line of code for id rather than increments so I replaced it so they should work with Okta it also would not let the foriegn key reference be done without doing it the other way as well....Good Eye

Reply:
Ahhh gotcha, I had no idea! Other than that sounds good!

Reply:
Glad to see the questioning. Good job @jonathan-portillo 

Solid answer @karapeoples